test/e2e/deployimage/plugin_cluster_test.go

```diff
@@ -263,6 +266,24 @@ func Run(kbc *utils.TestContext) {
 	}
 	EventuallyWithOffset(1, getMemcachedPodStatus, time.Minute, time.Second).Should(Succeed())
 
+	By("validating that the status of the custom resource created is updated or not")
+	var status string
+	var customResourceConditions metav1.Condition
+	//nolint:lll
+	getStatus := func() error {
+		status, err = kbc.Kubectl.Get(true, strings.ToLower(kbc.Kind),
+			strings.ToLower(kbc.Kind)+"-sample", "-o", "jsonpath={.status.conditions}")
+		if err == nil && strings.TrimSpace(status) == "" {
+			return errors.New("empty status, continue")
+		} else if err := goyaml.Unmarshal([]byte(status), &customResourceConditions); err != nil {
+			return errors.New("unable to Unmarshal the status")
+		} else if customResourceConditions.Type != "typeAvailable" || customResourceConditions.Status != metav1.ConditionTrue {
+			return errors.New(`conditions type should be "typeAvailable" and status should be true`)
+		}
+		return nil
+	}
+	Eventually(getStatus, time.Minute, time.Second).Should(Succeed())
```

Comment:
After we unmarshal we can get the result and do the check outside 
We should ensure that you have typeAvailable == true